Libraries/LibWebView/Process.cpp

```diff
@@ -50,8 +50,12 @@ ErrorOr<Process::ProcessAndIPCTransport> Process::spawn_and_connect_to_process(C
     return ProcessAndIPCTransport { move(process), IPC::Transport(move(ipc_socket)) };
 }
 
-ErrorOr<Optional<pid_t>> Process::get_process_pid(StringView process_name, StringView pid_path)
+ErrorOr<Optional<pid_t>> Process::get_process_pid([[maybe_unused]] StringView process_name, [[maybe_unused]] StringView pid_path)
 {
+    // FIXME: Implement WebView::Process::get_process_pid on Windows
+#ifdef AK_OS_WINDOWS
```

Comment:
I would rather ifdef out the implementation of both of these methods than have the ifdefs inside the method.